src/screens/OtherProfile/index.js

```diff
@@ -602,7 +610,7 @@ const OtherProfile = () => {
         {isShowButton ? (
           <TouchableNativeFeedback onPress={toTop}>
             <View style={{...styles.btnBottom, opacity}}>
-              <ArrowUpWhiteIcon width={12} height={20} fill={colors.white} />
+              <ArrowUpWhiteIcon width={12} height={20} fill={COLORS.white} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The fill color of the `ArrowUpWhiteIcon` has been updated to use `COLORS.white` from the `theme` module. This is a good practice for maintaining consistency across the application.



<!-- This is an auto-generated comment by OSS CodeRabbit -->